cmd/notation/common.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import "github.com/urfave/cli/v2"
+
+var (
+	usernameFlag = &cli.StringFlag{
+		Name:    "username",
+		Aliases: []string{"u"},
+		Usage:   "username for generic remote access",
+		EnvVars: []string{"NOTATION_USERNAME"},
+	}
+	passwordFlag = &cli.StringFlag{
+		Name:    "password",
+		Aliases: []string{"p"},
+		Usage:   "password for generic remote access",
+		EnvVars: []string{"NOTATION_PASSWORD"},
+	}
```

Comment:
Taking password from cli in plain text is a security risk as its visible while typing and can be seen in shell history.